brands/shop/clothes.json

```diff
@@ -3433,6 +3433,15 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|Windsor": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "Windsor",
+      "clothes": "women",
+      "name": "Windsor",
```

Comment:
We'll need a wikidata key here before this will show up in iD and other editors.

Reply:
I couldn't find a relevant Wikipedia/Wikidata entry for Windsor